packages/serverless-cdk-plugin/src/serverlessCdk.ts

```diff
@@ -122,12 +141,11 @@ export class ServerlessCdkPlugin implements Plugin {
 
   async instantiateConstruct(): Promise<void> {
     const serverlessConfigFile = await getServerlessConfigFile();
-    const MyConstruct = serverlessConfigFile.cdkConstruct;
+    const ServerlessCdkConstruct = serverlessConfigFile.serverlessConstruct;
 
-    this.construct = new MyConstruct(
-      this.stack,
-      'serverlessCdkBridgeConstruct',
-    );
+    this.construct = new ServerlessCdkConstruct(this.stack, 'cdk', {
+      serverless: serverlessConfigFile,
```

Comment:
dynamically check if we have a `ServerlessConstruct` and only give the props if we have. We want to stay compatible with simple `Construct`